python-libraries/data-platform-catalogue/data_platform_catalogue/entities.py

```diff
@@ -50,22 +66,39 @@ def from_data_product_metadata_dict(metadata: dict, version, owner_id: str):
             description=metadata["description"],
             version=version,
             owner=owner_id,
+            owner_display_name=metadata["dataProductOwnerDisplayName"],
+            maintainer=metadata.get("dataProductMaintainer"),
+            maintainer_display_name=metadata.get("dataProductMaintainerDisplayName"),
             email=metadata["email"],
+            status=DataProductStatus[metadata["status"]],
             retention_period_in_days=metadata["retentionPeriod"],
             domain=metadata["domain"],
             dpia_required=metadata["dpiaRequired"],
+            dpia_location=metadata.get("dpiaLocation"),
+            last_updated=datetime.strptime(metadata["lastUpdated"], "%Y%m%d"),
+            creation_date=datetime.strptime(metadata["creationDate"], "%Y%m%d"),
+            s3_location=metadata.get("s3Location"),
             tags=metadata.get("tags", []),
         )
 
         return new_metadata
 
 
+class DataSensitivityLevel(Enum):
```

Comment:
Could we call this `SecurityClassification` or `GovernmentSecurityClassification` or something like that? Just to reuse existing terminology and avoid confusion with the "sensitive" marking which we might also want to add

https://www.gov.uk/government/publications/government-security-classifications/government-security-classifications-policy-html#additional-markings